### DISCUSSION TITLE
not understand concept

### DISCUSSION THREAD

**Discussion by pintail120** [WLAN-Pi/feedback#47] [General Inquiries] [answered]
Dear all
I think this is a brilliant idea and I would of thought everyone would be building this but there is hardly anything recent on google searches and youtube, mostly out of date. I do not even see anyone 3D printing casings or uploading STL's ? 
its almost as if its a dead project ?
The only comprehensive website is with someone selling the complete unit for some astounding amount of £400
please help, confused

**Comment by nickjvturner** [comment, marked as answer]
Hi there, I am the someone selling complete M4 units.

homepage: https://www.wlanpi.com/
If you are interested to build your own R4, you can find the parts list here: https://userguide.wlanpi.com/hardware/wlan-pi-r4
download the model files here: https://www.printables.com/model/265668-wlan-pi-r4-fascia-essentials

**Comment by pintail120** [comment]
Hi NIck
thank you for your reply
can you please tell me where I can find the walk through for the self build
for the wlan pi pro ?
thanks & regards
Gavin

**Comment by pintail120** [comment]
Hi Nick
Could you please tell me why there limitations on M4 and R4 as opposed to
the wlan pi pro ?
Is the wlan pi pro a commercial version and the m4 and r4 open source
versions ?
Is there a self build option for the Pro ?
thanks & regards
Gavin

**Reply by nickjvturner** [reply]
Hi Gavin,

Here is a comparison chart with all the WLAN Pi units:
https://userguide.wlanpi.com/hardware/compare-wlan-pi-models

We only produced ~200 WLAN Pi Pro units, they are not generally available for purchase at this time. We do not know **if / when** we will manufacture more WLAN Pi Pro units.

There is not self build option for the Pro because it uses a custom carrier board, that we do not have stock of.
The R4 is simple to build yourself, you can acquire the parts by your self or you can buy a BYOP kit.
I hope to have R4 complete units for sale in the near future.

The WLAN Pi M4 is only available as a complete unit due to the relative complexity of assembly, that being said, you can find all the parts that we use in the M4. I do not wish to sell a 'kit' version of the M4 on my web store, but if you are keen to assemble your own we can discuss options for purchasing an M4 case.

Nick

**Comment by pintail120** [comment]
Hi Nick
I remember seeing a link for the custom carrier board on aliexpress ? is
this an option ?
When will the custom carrier board be in stock again ?
Where would I get all the parts for the M4 I am electronic engineer
therefore dont have issues working with SMD or intricate electronics ?
I also have a 3D printer and have printer all the parts for the R4, I will
making this to see how it does, can you please tell me why the R4 does not
have the functionality of the WLAN Pi pro ? is it a hardware limitation or
license ?
thanks & regards
Gavin

**Reply by nickjvturner** [reply]
The board from AliExpress is the carrier board which we built the WLAN Pi M4 on top.
We currently have no plans to manufacture more carrier boards for the WLAN Pi Pro.
The parts list for the M4 can be found in the User Guide, but we have not made a comprehensive guide for DIY WLAN Pi M4.
Parts mostly come from AliExpress.

All limitations of the various WLAN Pi models is due to hardware limitations.
